TBouder/RarityExtended
# Feature/Fix blank page

## What it does ✨
Fix the blank start for new users. You now have a welcome from Facu where you can choose your first adventurer !

![Capture d’écran 2021-09-11 à 12 56 56](https://user-images.githubusercontent.com/9974362/132945746-6935e779-9995-41e8-a079-f54ab9ad4245.png)

Should fix #16 

## How to test ✅
With an empty account, login on RE and try to Summon an adventurer, you should be able to do it and you should be redirected to the standard index page
